google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -274,10 +278,36 @@ class ConnectionWorker implements AutoCloseable {
           add(AttributeKey.stringKey("trace_field_3"));
         }
       };
+  static AttributeKey<String> telemetryKeyErrorCode = AttributeKey.stringKey("error_code");
   private Attributes telemetryAttributes;
+  private long incomingRequestCountBuffered;
```

Comment:
nit, I would group all of them under telemetryMetrics.

Reply:
Done